src/dswx_sar/save_mgrs_tiles.py

```diff
@@ -859,10 +870,18 @@ def run(cfg):
             landcover_mask=landcover_mask,
             bright_water_fill=bright_water_mask,
             dark_land_mask=dark_land_mask,
+            inundated_vegetation=(inundated_vege_high_ratio == 1) &
+                (wetland == 0) & (water_map == 0),
+            wetland_nonwater=(water_map == 0) & wetland,
+            wetland_water=(water_map == 1) & wetland,
+            wetland_bright_water_fill=bright_water_mask & wetland,
+            wetland_inundated_veg=(inundated_vegetation == 2) &
+                wetland,
```

Comment:
```suggestion
            wetland_inundated_veg=(inundated_vegetation == 2) & wetland,
```
No need to split the line here.